boards/stm32f3discovery/dist/debug.sh

```diff
@@ -1,4 +1,28 @@
 #!/bin/sh
 
-echo "Debugging $1"
+if [ ! -f "$2" ]; then
+    echo "ELF-file $2 does not exist"
+    exit 1
+fi
+
+echo "##"
+echo "## Debugging $2"
+echo "##"
+
+openocd -f "board/stm32f3discovery.cfg" \
+    -c "tcl_port 6333" \
+    -c "telnet_port 4444" \
```

Comment:
I would suggest specifying `gdb_port` as well here, and maybe make it depend on an environment variable, such as GDB_PORT or something which can be set by the Makefile.